src/component/MSWToolbar.tsx

```diff
@@ -111,41 +116,71 @@ export const MSWToolbar = ({
     set(prefix, 'delay', String(delay));
   }, [delay, prefix]);
 
+  const [isHidden, setIsHidden] = React.useState(false);
+
   if (!isEnabled || !worker) return <>{children}</>;
 
-  return isReady ? (
-    <>
-      <>
-        <div className={styles.container}>
-          <div className={styles['left-actions']}>
-            <div className={styles['input-wrapper']}>
-              <label>Mocks:</label>
+  if (!isReady) {
+    return null;
+  }
 
-              <div className={styles.onoffswitch}>
+  return (
+    <>
+      <button
+        className={`${styles.btn} ${styles['show-toolbar-button']}`}
+        onClick={() => {
+          setIsHidden(false);
+        }}
+        hidden={!isHidden}
+      >
+        <MSWLogo width={64} />
+      </button>
+
+      <div
+        className={[className, styles['msw-toolbar']].filter(Boolean).join(' ')}
+        data-hidden={isHidden}
+        data-position={position}
+        {...props}
+      >
+        <div>
+          <button
+            className={`${styles.btn} ${styles['close-button']}`}
+            onClick={() => {
+              setIsHidden(true);
+            }}
+          >
+            Close
+          </button>
```

Comment:
We should consider preserving this setting like the others so that if someone reloads it doesn't default to open once closed. We should also probably default to closed as well?